### PR TITLE
Fixing potential index out of bound exception

### PR DESCRIPTION
**Pull request by pgbhagat**

### CODE REVIEW

File: azure-media/src/main/java/com/microsoft/windowsazure/services/media/authentication/AzureAdTokenProvider.java [hunk 1]
```diff
@@ -86,7 +86,7 @@ public AzureAdAccessToken acquireAccessToken() throws Exception {
             default:
                 throw new NotImplementedException(
                     String.format(
-                        "Token Credential type %1 is not supported.",
```
Comment (pgbhagat) [inline azure-media/src/main/java/com/microsoft/windowsazure/services/media/authentication/AzureAdTokenProvider.java]: fixing incorrect String formatter arg

File: azure-media/src/main/java/com/microsoft/windowsazure/services/media/entityoperations/EntityRestProxy.java [hunk 2]
```diff
@@ -196,7 +196,7 @@ public String update(EntityUpdateOperation updater) throws ServiceException {
         updater.processResponse(clientResponse);
         if (clientResponse.getHeaders().containsKey("operation-id")) {
             List<String> operationIds = clientResponse.getHeaders().get("operation-id");
-            if (operationIds.size() >= 0) {
```
Comment (pgbhagat) [inline azure-media/src/main/java/com/microsoft/windowsazure/services/media/entityoperations/EntityRestProxy.java]: if the operationIds.size is ZERO, then operationIds.get(0) will throw index out of bound exception. Fixing it

File: azure-media/src/main/java/com/microsoft/windowsazure/services/media/entityoperations/EntityRestProxy.java [hunk 3]
```diff
@@ -219,7 +219,7 @@ public String delete(EntityDeleteOperation deleter) throws ServiceException {
         PipelineHelpers.throwIfNotSuccess(clientResponse);
         if (clientResponse.getHeaders().containsKey("operation-id")) {
             List<String> operationIds = clientResponse.getHeaders().get("operation-id");
-            if (operationIds.size() >= 0) {
```
Comment (pgbhagat) [inline azure-media/src/main/java/com/microsoft/windowsazure/services/media/entityoperations/EntityRestProxy.java]: if the operationIds.size is ZERO, then operationIds.get(0) will throw index out of bound exception. Fixing it

File: azure-media/src/main/java/com/microsoft/windowsazure/services/media/entityoperations/EntityRestProxy.java [hunk 4]
```diff
@@ -269,7 +269,7 @@ public String action(EntityActionOperation entityActionOperation)
         
         if (clientResponse.getHeaders().containsKey("operation-id")) {
             List<String> operationIds = clientResponse.getHeaders().get("operation-id");
-            if (operationIds.size() >= 0) {
```
Comment (pgbhagat) [inline azure-media/src/main/java/com/microsoft/windowsazure/services/media/entityoperations/EntityRestProxy.java]: if the operationIds.size is ZERO, then operationIds.get(0) will throw index out of bound exception. Fixing it

File: azure-media/src/main/java/com/microsoft/windowsazure/services/media/models/StreamingEndpoint.java [hunk 5]
```diff
@@ -105,7 +105,7 @@ public Object processResponse(Object rawResponse) throws ServiceException {
 
             if (clientResponse.getHeaders().containsKey("operation-id")) {
                 List<String> operationIds = clientResponse.getHeaders().get("operation-id");
-                if (operationIds.size() >= 0) {
```
Comment (pgbhagat) [inline azure-media/src/main/java/com/microsoft/windowsazure/services/media/models/StreamingEndpoint.java]: same as above comment

**Comment by JonathanGiles** [comment]
@praries880 Please review and merge / close this PR as appropriate. Thanks.

**Comment by JonathanGiles** [comment]
This PR has conflicting files. @pgbhagat Did you want to update this PR, or should we close it? Thanks!

**Comment by pgbhagat** [comment]
> 
> 
> This PR has conflicting files. @pgbhagat Did you want to update this PR, or should we close it? Thanks!

Let me check

**Comment by jianghaolu** [comment]
Just FYI @pgbhagat  The media service SDK has been moved to https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/mediaservices/microsoft-azure-media. Once you apply your changes there we will be able to move forward. Thanks!

**Comment by joshfree** [comment]
Closing this old PR that needs updates to match the directory structure changes to the repo.  As others have said, once the changes are applied we'll be able to move forward with merging the PR.

Feel free to re-open once the changes are made.  Thank you

**Pull request closed without merging**